tests/typecheck/topol/Cast.java

```diff
@@ -0,0 +1,12 @@
+package typecheck.topol;
+
+import universe.qual.Peer;
+import universe.qual.Rep;
+
+public class Cast {
+    void foo() {
+        @Peer Object p;
+        // :: warning: (cast.unsafe)
```

Comment:
Shouldn't this be an error? Peer and rep are not comparable, so this cast will always fail.